CODEOWNERS

```diff
@@ -1,2 +1,7 @@
 # Keystone
 /pkg/values/ @cedric-cordenier
+
+# CCIP
+/pkg/types/ccip @smartcontractkit/ccip
+/pkg/types/ccipocr3 @smartcontractkit/ccip
+/pkg/types/provider_ccip* @smartcontractkit/ccip
```

Comment:
Could you delete this file and then use CODEOWNERS in the `.github` repo? We want to make that the default location going forward since github looks there first (cc @jmank88)